tests/ArmTemplates.Tests/Extractor/Scenarios/GatewayApiExtractorTests.cs

```diff
@@ -76,5 +78,116 @@ public async Task GenerateGatewayApisTemplates_ProperlyLaysTheInformation()
             gatewayResources.All(x => x.Type == ResourceTypeConstants.GatewayApi).Should().BeTrue();
             gatewayResources.All(x => x.Properties is not null).Should().BeTrue();
         }
+
+        [Fact]
+        public async Task GenerateGatewayApisTemplates_ProperlyLaysTheInformation_GivenApiNameParameterIsGiven()
```

Comment:
What do you mean with ProperlyLaysTheInformation?

Reply:
updated the name, thank you.